content/blog/adoptium-reproducible-builds/index.md

```diff
@@ -0,0 +1,107 @@
+contributed back upstream to the OpenJDK community project. These fixes have also been backported to the
+jdk17u stream, since jdk-17 is a LTS release.
+
+OpenJDK contributions:
+- [8284539: Configure --with-source-date=version fails on MacOS](https://github.com/openjdk/jdk/pull/8247)
+- [8284661: Reproducible assembly builds without relative linking](https://github.com/openjdk/jdk/pull/8177)
+- [8284437: Building from different users/workspace is not always deterministic ](https://github.com/openjdk/jdk/pull/8124)
+- [8283315: jrt-fs.jar not always deterministically built](https://github.com/openjdk/jdk/pull/7852)
+- [8279834: Alpine Linux fails to build when --with-source-date enabled](https://github.com/openjdk/jdk/pull/7025)
+- [8279182: MakeZipReproducible ZipEntry timestamps not localized to UTC](https://github.com/openjdk/jdk/pull/6926)
+- [8278766: Enable OpenJDK build support for reproducible jars and jmods using --date](https://github.com/openjdk/jdk/pull/6878)
+- [8278163: --with-cacerts-src variable resolved after GenerateCacerts recipe setup](https://github.com/openjdk/jdk/pull/6680)
+- [8278080: Add --with-cacerts-src='user cacerts folder' to enable deterministic cacerts generation](https://github.com/openjdk/jdk/pull/6647)
+- [8277762: Allow configuration of HOTSPOT_BUILD_USER](https://github.com/openjdk/jdk/pull/6542)
+- [8276766: Enable jar and jmod to produce deterministic timestamped content](https://github.com/openjdk/jdk/pull/6481)
+- [8276764: Enable deterministic file content ordering for Jar and Jmod](https://github.com/openjdk/jdk/pull/6395)
+- [8276743: Make openjdk build Zip Archive generation "reproducible"](https://github.com/openjdk/jdk/pull/6311)
+- [8276654: element-list order is non deterministic](https://github.com/openjdk/jdk/pull/6278)
+
+### Ongoing Eclipse Adoptium Reproducible Build projects
+
+As well as continuing to identify and fix any OpenJDK non-deterministic issues, Eclipse Adoptium is continuing to
+integrate changes into the build scripts to fully support Reproducible Builds.
+
+The following capabilities are available at Eclipse Adoptium:
+
+- Reproducible jdk-19 builds for Linux architectures x64, aarch64, ppc64, arm32, and s390x.
+
+The following are projects currently in-progress:
+
+- Reproducible jdk-19 builds for MacOS x64 and MacOS aarch64.
+- Further non-deterministic issue resolutions, and subsequent contribution to upstream OpenJDK, and backporting to jdk17u.
+- Enhancing the Adoptium jenkins pipelines to integrate CycloneDX SBOM tooling to provide a standardized bill of materials framework for the Adoptium binaries.
+- Detailed dependency bill of materials analysis to fully extended the CycloneDX SBOM json.
+
+Future projects:
+
+- Extending Temurin reproducible build script support to the jdk17u build pipelines.
+- Further exploring required fixes to non-deterministic build output on other platforms, eg.Windows.
```

Comment:
```suggestion
- Further exploring required fixes to non-deterministic build output on other platforms, e.g.Windows.
```
I'd be tempted to call out Windows explicitly as something we intend to persue for definite, and maybe mention others in a separate bullet point, as it would be particularly good to encourage anyone interested in e.g. AIX to step forward and help :-) (I might talk to Tyler explicitly about that one)